containership/csctl
# Refactor clients/clientset to remove viper dependency

### Description

 #### What does this pull request accomplish?

 We need to allow consumers of the client to pass in granular configs instead of having a `viper` dependency within.

 #### What issue(s) does this fix?
* No issue

 #### Additional Considerations

 ### Testing

 - See that API + Provision service base URL override still works via config
 - See that API + Provision service base URLs are still properly defaulted without override

 ### Dependencies
* None
